### PR TITLE
fix batchsink encrytion widget spec

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: widgets/S3-batchsink.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
           "widget-attributes": {
             "layout": "inline",
```
Comment (ajainarayanan) [inline widgets/S3-batchsink.json]: nit: Do we support this property? Was this added newly?

**Review by ajainarayanan — APPROVED** [review]
Minor comment. 👍 LGTM

File: widgets/S3-batchsink.json (hunk 1 above)
Comment (elfenheart) [reply widgets/S3-batchsink.json]: will remove

Comment (itsanudeep) [reply widgets/S3-batchsink.json]: @elfenheart @ajainarayan We have this property in radio group widget as well to display the buttons as inline or as block elements. Let me know if I have to change it there as well ( I'm assuming we use it)

**Pull request merged**